client/src/components/utilities.ts

```diff
@@ -11,12 +11,27 @@ export const tryJson = (responseText: string) => {
   }
 }
 
+const parseCookies = () =>
+  Object.fromEntries(
+    document.cookie.split(';').map(pair => pair.trim().split('='))
+  )
+
+export const addCSRFToken = (options?: RequestInit) => {
+  const token = parseCookies()._csrf_token
+  if (token && options && ['POST', 'PUT', 'DELETE'].includes(options.method!))
+    return {
+      ...options,
+      headers: { ...options.headers, 'X-CSRFToken': token },
```

Comment:
I recently learned that `X-` for HTTP headers is the deprecated naming scheme, and more accepted is something like `anti-csrf-token` or the like. I don't care *that* much, but thought I'd mention it in case you want to do the more modern way.

Reply:
I'm fine with just using the library default here I think